### DISCUSSION TITLE
Question on importing CSV records

### DISCUSSION THREAD

**Discussion by Ashok0** [mindsdb/mindsdb#2343] [Get help] [answered]
Hi,

I have a question regarding usage of imported data with MindsDB.  I registered a MindsDB account and selected "Add Data" > "Import File" and uploaded some sample CSV data to a table.   I can view the data just fine as shown in the following screencap:  https://imgur.com/a/X6mrzHt

If I attempt to select an individual column i.e. 'sensitivity' with the following statement:  **SELECT sensitivity FROM files.bedcheck**, 
I receive the following error message:  **Binder Error: Referenced column "sensitivity" not found in FROM clause! Candidate bindings: "df_table. sensitivity" LINE 1: SELECT sensitivity AS sensitivity**

Any assistance in how I can resolve this error would be greatly appreciated.  

Thank you!

**Comment by ZoranPandovski** [comment]
Hi @Ashok0 I just tried this with another file and it works for me. Would it be possible for you to share the dataset so I can replicate the issue?

**Comment by Ashok0** [comment]
Hi, thanks for the reply!  Here is my [CSV file](https://drive.google.com/file/d/1xTdg3XY22nLjqfz9S2kI7RUqjOU4RV7f/view?usp=sharing)

The first column is named 'p_id' and the following statement works:  _SELECT p_id FROM files.bedcheck LIMIT 10;_

The second column is named 'mac_address', but the following statement:  _SELECT mac_address FROM files.bedcheck LIMIT 10;_

results in the following error:   **Binder Error: Referenced column "mac_address" not found in FROM clause!
Candidate bindings: "df_table. mac_address"
LINE 1: SELECT mac_address AS mac_address**

**Comment by ZoranPandovski** [comment, marked as answer]
Looking at the error message `Candidate bindings: "df_table. rval", "df_table.peak", "df_table.p_id", "df_table.diff", "df_table.sent"` it looks like somehow we didn't parse the empty space in the header so ` sensitivity` can't be found. For now, can you please just exclude the empty space and I will post an issue so we can strip() headers.

**Reply by ZoranPandovski** [reply]
Also just try: SELECT ` mac_address` FROM files.bedcheck1 LIMIT 10; with empty space after  `  column_name`

**Reply by Ashok0** [reply]
Thanks, simply removing the whitespace has fixed the issue.  I should be all set, thank you again!
